rust/main/agents/validator/src/validator.rs

```diff
@@ -53,6 +53,7 @@ pub struct Validator {
     chain_metrics: ChainMetrics,
     runtime_metrics: RuntimeMetrics,
     agent_metadata: AgentMetadata,
+    announced: IntGauge,
```

Comment:
I suppose that validator will set the metric once when it (re)-started. I wonder if we really need it as a field. We could have just a method which gets the metric from `core_metrics` and sets it.